### ISSUE TITLE
Provide an output format as "Sonar Generic Issue Import format",

### ISSUE THREAD

**Issue by saeeda1981** [hadolint/hadolint#626]
- [] This is a bug report
- [x ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Can you provide an output format as "Sonar Generic Issue Import format", this will help the output to be consumed directly into Sonarqube

https://docs.sonarqube.org/latest/analysis/generic-issue/
 
### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
(paste your output here)
```

**Dockerfile** (if relevant)

```Dockerfile
(paste content of your Dockerfile)
```

**Additional environment details (OS, stack version, etc.)**

**Comment by lorenzo** [comment]
would you like to contribute this feature?
